store/postgres/src/store.rs

```diff
@@ -365,6 +368,12 @@ impl Store {
                 ValueType::ID => "",
                 ValueType::Int => "::bigint",
                 ValueType::String => "",
+                ValueType::List => {
+                    return Err(QueryExecutionError::OrderByNotSupportedError(
+                        "List".to_string(),
+                        "All".to_string(),
```

Comment:
`All` doesn't make sense to me. Perhaps we should add a different `QueryExecutionError` variant here, like `OrderByNotSupportedForType("List".to_string())`.